solaris/solaris11/generate_wazuh_packages.sh

```diff
@@ -298,17 +283,33 @@ ctrl_c() {
 
 
 show_help() {
-  echo
-  echo "Usage: $0 [OPTIONS]"
-  echo
-  echo "    -b, --branch <branch>               Select Git branch or tag e.g. $wazuh_branch."
-  echo "    -e, --environment                   Install all the packages necessaries to build the pkg package."
-  echo "    -s, --store  <pkg_directory>        Directory to store the resulting pkg package. By default, an output folder will be created."
-  echo "    -p, --install-path <pkg_home>       Installation path for the package. By default: /var."
-  echo "    -c, --checksum                      Compute the SHA512 checksum of the pkg package."
-  echo "    -h, --help                          Shows this help."
-  echo
-  exit $1
+    echo -e ""
+    echo -e "NAME"
+    echo -e "        $(basename $0) - Generate a Solaris 11 package"
+    echo -e ""
+    echo -e "SYNOPSIS"
+    echo -e "        $(basename $0) [OPTIONS]"
+    echo -e ""
+    echo -e "DESCRIPTION"
+    echo -e "        -b, --branch <branch>"
```

Comment:
Same

Reply:
Fixed in 51c6056cd937c0017172a0571a404418f91c9bb2